.github/issue_template.md

```diff
@@ -1,48 +1,29 @@
-Note: This is just a template, so feel free to use/remove the unnecessary things
-
+### Issue request type
+
+<!-- 
+    Required
+    Please add only one X to one of the following types. Do not fill multiple types (split the issue otherwise)
+    Please note this is not a GitHub task list, indenting the boxes or changing the format to add a '.' or '*' in front
+    of them would change the meaning incorrectly. The only changes to be made are to add a description text under the
+    description heading and to add a 'x' to the correct box.
+
+    [X] Question
+    [ ] Enhancement 
+    [ ] Bug 
+-->
+
+[ ] Question  
+[ ] Enhancement  
+[ ] Bug  
+
+<!-- 
+    Required
+    Add detailed description of what is being reported
+    Good example: https://os.mbed.com/docs/latest/reference/workflow.html
+    Things to consider sharing:
+    - What target does this relate to?
+    - What toolchain is being used?
+    - What is the SHA of Mbed OS (git log -n1 --oneline)?
+    - Steps to reproduce (Did you publish code or a test case that exhibits the problem)
+-->
 ### Description
```

Comment:
For consistency with the PR template, Description should come first at the top ?

Reply:
I think I'm actually fine as-is. Issues tend to need a lot more detail than PRs. Knowing the type of issue before the description helps frame the mindset before reading the description.

Who knows, maybe the PR template could be modified to have the type come before the description?

Reply:
Open for suggestions, could also move PR type first.

Reply:
I'm not really seeing an issue with the original template, or a need to align particularly. PRs and issues are different things. Maybe change some things to checkboxes for a stylistic alignment, but structually this has always seemed fine to me.

Reply:
Lot of people ignore it (either remove it or provide default values). Similar what we experienced with PR template.